cmd/nsdns/watch.go

```diff
@@ -35,43 +35,48 @@ func watchCommand() *cobra.Command {
 			}
 
 			dm.RefreshesCacheOnUpdate = true
-			if err := dm.UpdateCache(); err != nil {
+
+			clientset, err := GetKubernetesClientSet()
```

Comment:
Makes a bit of sense to load up the local stuff before hitting WAN, in case this fails, why bother ever reaching out?